iops_test.go

```diff
@@ -48,47 +48,22 @@ func TestGetValues(t *testing.T) {
 	for name, tt := range cases {
 		t.Run(name, func(t *testing.T) {
 			server := httptest.NewServer(&tt.fakeHandler)
-			go getValues(server.URL, tt.queryType)
+			URL = server.URL
+			go getValues(tt.queryType)
```

Comment:
Can you explain me here what this test does? (As you've not mentioned in the commit message) Also, Do we need `go` routine in the test? :thinking:

Reply:
yes, we need go routine because the function that we are testing returns the output in channel.

Reply:
I think, first you should write a test which tests `getValues` method first.